parsl/dataflow/dflow.py

```diff
@@ -773,6 +777,8 @@ def first(t):
     def add_executors(self, executors):
         for executor in executors:
             executor.run_dir = self.run_dir
+            executor.hub_address = self.hub_address
+            executor.hub_port = self.hub_interchange_port
```

Comment:
can you add a note to parsl.providers.base that this property exists?

Reply:
I am not too sure that I understood -- why it goes to `parsl.providers.base` but not executor?
Do you have an example for that? This is actually a property automatically set by DFK, not set by user.

Reply:
you are right: executor base not provider base.

It should be documented there because some other people will want to implement an executor, or understand how executors will be driven by the DFK.

That might be "users" or it might be "parsl developers": workflow users probably won't call any methods on an executor at all.

`run_dir` is an example of a similar property that holds some DFK context that the executor runs with; or the `label` field. They are documented in different ways in `parsl.executor.base`

Reply:
I have added them to `parsl.executor.base`. See if that is what you expect.